lib/scrabble.rb

```diff
@@ -1,7 +1,36 @@
 class Scrabble
 
+  def standardized_format(word)
+    word.upcase
+  end
+
   def score(word)
-    1
+    return 0 if word.nil?
+
+    values = point_values
```

Comment:
You can just call point_values wherever it's needed instead of assigning it to a variable here.